Web/starter_project_vanguard/components/layout/Navbar.tsx

```diff
@@ -0,0 +1,82 @@
+import clsx from 'clsx'
+import Image from 'next/image'
+import Link from 'next/link'
+import { AiOutlineMenu } from 'react-icons/ai'
+import { navItems } from './data/nav-items'
+import { useState } from 'react'
+import logo from '../../public/img/a2sv-logo.png'
+
+
+const Navbar:React.FC = () => {
+  const [toggle, setToggle] = useState(false)
+  const [active, setActive] = useState("/")
+  const setActiveAndToggle = (current:string)=>{
+    setActive(current)
+    setToggle(!toggle)
+  }
+  return (
+    <div className="sticky top-0 z-50 bg-white bg-opacity-100 flex justify-between font-montserrat px-10 pt-5">
```

Comment:
[check here](https://fonts.google.com/specimen/Montserrat)
As you can see from the above link, the Montserrat font does not have serifs; I'm not sure why the font you used did. Please validate that the font you used and the UI's visual appearance are the same.